main.tf

```diff
@@ -17,8 +13,4 @@ resource "mcaf_aws_account" "default" {
     lastname  = var.sso_lastname
     email     = var.sso_email
   }
-
-  lifecycle {
-    prevent_destroy = true
-  }
```

Comment:
Can we now call this on a molule? Otherwise I'm not sure if we should remove these...

Reply:
Sadly not yet, I had hoped it could be passed to the module or set via a variable but alas...

With these in place it's impossible to update an email alias or fix a failed Service Catalog provisioning attempt: 

- In the event an `mcaf_o365_alias` resource is destroyed, the address can just be added back with no impact
- In the event an `mcaf_aws_account` resource being destroyed, it's not great but the the account is "evicted" from Control Tower and could be enrolled manually using the console and then the consumer has the inconvenience of dealing with an empty `id` output...

Reply:
Hmm... In doubt a bit... As we would like to be able to prevent this from happening by accident and if we cannot do that on the module, then I don't see how else we can achieve that.

Reply:
I can understand that, but one could argue you'd want to prevent any accidental deletion of a resource... this is why there is a plan and review stage right?

I think at a minimum we should be fine to remove the lifecycle block from the `mcaf_o365_alias` resource.

Reply:
I'm fine with removing them I guess...